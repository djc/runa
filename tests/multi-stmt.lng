def __main__(name: str, args: list[str]) -> int:
	print('Asara')
	print(str(16384))
	print('thunderstorm')
