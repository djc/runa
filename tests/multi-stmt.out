Asara
16384
thunderstorm
